exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -77,7 +77,9 @@ def initialize(endpoint: nil,
         def export(span_data, timeout: nil)
           return FAILURE if @shutdown
 
-          send_bytes(encode(span_data), timeout: timeout)
+          send_bytes(
+            measure_encode_duration { encode(span_data) },
```

Comment:
I think we should inline `measure_encode_duration` into the `encode` method. We didn't do that with `measure_request_duration` because the block it wraps is a stdlib method (`@http.request(request)`).

Reply:
Sure thing